apps/payments/src/payments.service.ts

```diff
@@ -49,10 +61,16 @@ export class PaymentsService {
       currency,
     });
 
-    this.notificationsService.emit('notify_email', {
-      email,
-      text: `Your payment of ${formatter.format(amount)} has completed successfully.`,
-    });
+    if (!this.notificationsService) {
```

Comment:
We must explicitly call the onModuleInit here, otherwise this.notificationsService is not defined. I wonder why that is the case?